frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -342,18 +249,89 @@ export const PrometheusMultilineUtilizationItem = withDashboardResources<
   },
 );
 
-export const UtilizationCard = () => {
+const UtilizationCardNodeFilter: React.FC<UtilizationCardNodeFilterProps> = ({
+  machineConfigPools,
+  onNodeSelect,
+  selectedNodes,
+}) => {
   const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const sortedMCPs = machineConfigPools.sort((a, b) => {
+    const order = ['worker', 'master'];
+    if (order.indexOf(a.metadata.name) === -1) {
+      return 1;
+    }
+    if (order.indexOf(b.metadata.name) === -1) {
+      return -1;
+    }
+    return order.indexOf(a.metadata.name) - order.indexOf(b.metadata.name);
```

Comment:
This looks like it will sort two values not in the `order` array in a random order.

Reply:
Is there a better way to approach?  I Googled for this solution as I didn't know how to solve otherwise.

Reply:
I think this works (untested). You just need to handle the case where neither value is in the `order` array.

```ts
const indexA = order.indexOf(a.metadata.name);
const indexB = order.indexOf(b.metadata.name);
if (indexA === -1 && indexB === -1) {
  return a.metadata.name - b.metadata.name;
}
if (indexA === -1) {
  return 1;
}
if (indexB === -1) {
  return -1;
}
return indexA - indexB;
```

Reply:
Thanks, @spadgett.  I updated and tested.  It seems to work.
<img width="211" alt="Screen Shot 2021-06-15 at 11 29 39 AM" src="https://user-images.githubusercontent.com/895728/122081842-76cbf580-cdcd-11eb-8568-7037a5ae12e3.png">